pkgs/development/python-modules/python-lorem/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+}:
+
+buildPythonPackage rec {
+  version = "1.3.0.post1";
+  pname = "python-lorem";
+  disabled = pythonOlder "3.5";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-aokLCuQq6iHpC90MLCcIQxeEArPyx1o6RU1224xZdxY=";
+  };
+
+  buildInputs = [
+    setuptools
+  ];
+
```

Comment:
Please execute the tests https://github.com/JarryShaw/lorem/blob/master/test_lorem.py